src/components/icon/icons/medicalKit.jsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Icon from '../icon';
+
+export default function MedicalKit({
+  stroke,
+  fill,
+  strokeWidth,
+  style: styleProp,
+  ...rest // eslint-disable-line comma-dangle
+}) {
+  const style = {
+    ...styleProp,
+    ...Icon.defaultProps.style,
+  };
+  return (
+    <svg style={style} {...rest} viewBox={'0 0 20 17'}>
```

Comment:
* Maybe rename to `Medkit`? since that is a common abbreviation of the word
* Move `viewBox` to defaultProps
* Combine the `g` tags
* Remove `stroke` from propTypes and from the arguments list since it is not used